src/v2/view-builder/views/authenticator/SelectAuthenticatorUnlockAccountView.js

```diff
@@ -0,0 +1,16 @@
+import BaseView from '../../internals/BaseView';
+import BaseForm from '../../internals/BaseForm';
+import { loc } from 'okta';
+
+const UnlockAccountView = BaseForm.extend({
+  noButtonBar: true,
+  title: () => {
+    return loc('unlockaccount', 'login');
+  }
+});
+
+export default BaseView.extend({
+  initialize () {
+    this.Body = UnlockAccountView;
+  }
```

Comment:
Can we instead do a ```Body: UnlockAccountView``` which would override the `Body` from `BaseView` ? This would make sure the `initialize` from `BaseView` or `View` (extended by `BaseView`) gets called.

Or alternatively, ```BaseView.prototype.initialize.apply(this, arguments);``` can be added to achieve the same.

Reply:
I am doing this because incase if we have to flip the view based on some condition in future then you can do it here..

Reply:
Fixed with alternative approach you mentioned.